# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome24/straw/python-version.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- straw-0.19.1/Makefile.orig	2003-09-30 21:45:17.000000000 +0200
+++ straw-0.19.1/Makefile	2003-09-30 21:45:26.000000000 +0200
@@ -1,4 +1,4 @@
-PYTHON ?= python2.2
+PYTHON ?= python2.3
 INSTALL ?= install
 RM ?= rm
 MSGFMT ?= msgfmt
@@ -10,7 +10,7 @@
 # autodetect GNOME prefix, change this if you want it elsewhere
 PREFIX ?= `pkg-config libgnome-2.0 --variable=prefix || echo /usr/local`
 
-LIBDIR = $(PREFIX)/lib/python2.2/site-packages/straw
+LIBDIR = $(PREFIX)/lib/python2.3/site-packages/straw
 BINDIR = $(PREFIX)/bin
 DATADIR = $(PREFIX)/share/straw
 APPLICATIONSDIR = $(PREFIX)/share/applications
